default.nix

```diff
@@ -150,20 +150,52 @@ let
           inherit pyproject poetrylock overrides python pwd preferWheels;
         }
       );
-      editablePackages = lib.mapAttrsToList
-        (name: src:
-          # Creates a "fake" python module that just points to the editable source directory
-          # See https://docs.python.org/3.8/library/site.html for info on such .pth files
-          py.python.pkgs.toPythonModule (pkgs.runCommandNoCC "${name}-editable"
-            { } ''
-            mkdir -p "$out/${py.python.sitePackages}"
-            echo "${toString src}" > "$out/${py.python.sitePackages}/${name}.pth"
-          ''
-          )
-        )
-        editablePackageSources;
+
```

Comment:
Could you split this to a separate file?

Reply:
Like this?

Reply:
Nice, thanks!